packages/common/src/paths.tsx

```diff
@@ -0,0 +1,28 @@
+export const PATH_HOME = '/';
+
+export const PATH_MEDIA = '/m/:id/*';
+export const PATH_PLAYLIST = '/p/:id/*';
+export const PATH_LEGACY_SERIES = '/s/:id/*';
+
+export const PATH_SEARCH = '/q/*';
+export const PATH_ABOUT = '/o/about';
+
+export const PATH_USER = '/u/*';
+export const PATH_USER_ACCOUNT = '/u/my-account';
```

Comment:
I think building paths is a little easier to follow than splitting paths
```suggestion
export const PATH_USER_BASE = '/u'
export const PATH_USER = `${PATH_USER_BASE}/*`;
export const PATH_USER_ACCOUNT = `${PATH_USER_BASE}/${RELATIVE_PATH_USER_ACCOUNT}`;
```